### ISSUE TITLE
[NPC] Bloodsail Mage not casting before reaching melee range

### ISSUE THREAD

**Issue by phonkala** [SallyWhitemane/Maelstrom#438]
While I was testing another issue (https://github.com/SallyWhitemane/Maelstrom/issues/437) with caster mobs, ran into this:

Bloodsail Mages only start casting after they reach melee range. Tested this ~10 times and each time, only after they ran to melee range and hit me once, they started casting fireballs/novas. Of course they should be casting right away when aggroed.

**Comment by Afgann** [comment]
fixed
